CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Improved several warning/error messages to (more accurately) report the location (filename, line range) in which the warning/error originated. ([#2426], [#2752], [#2793], [#2803], [#2809])
 * Warn/error if jldoctest starts with an empty line; or lacks empty line between two REPL prompts. ([#2031], [#2083], [#2679], [#2808])
 * The `forcepush=true` option to `deploydocs` now uses `--force-with-lease` instead of `--force`. ([#2817])
+* Improved hashing in `HTMLWriter` to use `mdflatten` instead of trying to compute the hash of the Julia native node structure. ([#2710])
```

Comment:
I am not sure this helpful for end users, as they don't know what is being hashed. Better to explicitly mention admonitions, and also what the hash is being used for? For example
```suggestion
* Improved the generation of anchors for admonitions to be more stable and e.g. not change due to changes of the internal representation across different Julia versions. ([#2710])
```

Reply:
```suggestion
* Improved the generation of anchors for admonitions in HTML output to be more stable and e.g. not change due to changes of the internal representation across different Julia versions. ([#2710])
```